test/e2e/aws_test.go

```diff
@@ -694,30 +703,140 @@ func TestAWSFinalizerDeleteS3Bucket(t *testing.T) {
 	if err != nil {
 		t.Errorf("unable to get image registry resource: %#v", err)
 	}
-	driver, err := storage.NewDriver(&cr.Spec.Storage, kcfg, mockLister)
+
+	// Create an AWS config using the in-cluster credentials so that we can watch the S3 bucket
+	imageRegistryPrivateConfiguration, err := te.Client().Secrets(defaults.ImageRegistryOperatorNamespace).Get(
```

Comment:
@joelddiaz this test fails, you are trying to access this Secret after the operator's config is deleted (at this point the operator is finalizing the config resource and deleting dependant resources).

Reply:
@dmage IIUC, I suppose I can move grabbing the Secret's content above line 693 where the Configs().Delete() is called.